cli/api/daemon.go

```diff
@@ -713,15 +713,23 @@ func (d *daemon) startAgent() error {
 func (d *daemon) registerMasterRPC() error {
 	glog.V(0).Infoln("registering Master RPC services")
 
-	if err := d.rpcServer.RegisterName("Master", master.NewServer(d.facade)); err != nil {
+	server := master.NewServer(d.facade)
+	disableLocal := os.Getenv("DISABLE_RPC_BYPASS")
+	if disableLocal == "" {
+		rpcutils.RegisterLocalAddress(options.Endpoint, fmt.Sprintf("localhost:%s", options.RPCPort),
```

Comment:
maybe a log message here to record the fact that we're running in a non-default configuration